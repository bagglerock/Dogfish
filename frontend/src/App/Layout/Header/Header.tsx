import React from 'react';

export const Header: React.FC = () => (
  <header className="header p-3 text-center">
    <h1>Hangman</h1>
  </header>
);
