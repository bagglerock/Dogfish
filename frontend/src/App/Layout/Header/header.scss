.header {
  border-bottom: solid white 2px;

  h1 {
    font-size: 40px;
  }
}
