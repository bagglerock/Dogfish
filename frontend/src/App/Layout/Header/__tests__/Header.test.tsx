import { render } from '@testing-library/react';
import { Header } from 'app/Layout/Header/Header';
import React from 'react';

test('renders App/Layout', () => {
  const { container } = render(<Header />);

  expect(container).toMatchSnapshot();
});
