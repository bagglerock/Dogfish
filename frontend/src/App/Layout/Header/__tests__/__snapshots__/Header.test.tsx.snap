// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App/Layout 1`] = `
<div>
  <header
    class="header p-3 text-center"
  >
    <h1>
      Hangman
    </h1>
  </header>
</div>
`;
